packages/cascara/src/ui/Form/Form.js

```diff
@@ -178,6 +180,10 @@ const Form = ({
     !isEditable ? false : isInitialEditing
   );
 
+  const isInvalidFormState = !dataDisplay && !isInitialEditing;
```

Comment:
@rociobaena I think we could make some of these names more clear. The semantics of our name here should make sense when we use it below in the developer message warning. Why is the state invalid? We do not know until we go back and look at where the const is defined. Maybe something like this with also a comment would be more clear?
```suggestion
  // A form cannot be in an initial editing state while also not having a dataDisplay defined
  const isEditingWithoutDataDisplay = !dataDisplay && !isInitialEditing;
```

Reply:
Agree on @brianespinosa . Adding the comment.